### ISSUE TITLE
SquizMultiLineFunctionDeclaration error when param has trailing comment

### ISSUE THREAD

**Issue by gsouf** [squizlabs/PHP_CodeSniffer#1959]
Hi,

I'm encountering issue with ``Squiz.Functions.MultiLineFunctionDeclaration.OneParamPerLine`` snif.

The following works perfectly:

```php
    public function __construct(
        $foo,
        $bar
    ) {
```

while the following wont:

```php
    public function __construct(
        $foo,   // This is foo
        $bar
    ) {
```

and triggers ``error - Multi-line function declarations must define one parameter per line (Squiz.Functions.MultiLineFunctionDeclaration.OneParamPerLine)``

Please note that the following is working:

```php
    public function __construct(
        $foo /* this is foo */,
        $bar // this is bar
    ) {
```

**Comment by gsherwood** [comment]
I think it wants the comma to be the last thing on the line, but comments should be ignored by this check.

**Comment by jrfnl** [comment]
@gsouf @gsherwood Had a quick look and turned out to be a very simple fix ;-) See PR #1965
